sv_mini_atlas/library/importers.py

```diff
@@ -134,8 +134,7 @@ def destructure_node(self, node_urn, tokens):
 
     def generate_branch(self, line):
         ref, tokens = line.strip().split(maxsplit=1)
-        _, passage = ref.split(".", maxsplit=1)
-        node_data = self.destructure_node(f"{self.urn}{passage}", tokens)
+        node_data = self.destructure_node(f"{self.urn}{ref}", tokens)
```

Comment:
I don't see a problem with us standardizing (and documenting via https://github.com/scaife-viewer/scaife-viewer.org/wiki/ATLAS-Text-File-Structure) that the first portion of the line up to the first space is what we'll use as the ref.

Do you agree @jtauber?  I think that the Iliad and the Odyssey  have just "lagged" behind the format used for the other texts because they were the first ones.